src/components/Tag/Tag.js

```diff
@@ -59,6 +50,19 @@ const tagClickableStyles = ({ onClick, theme }) =>
       ${shadowBorder(theme.colors.n300)};
     }
   `;
+const tagSelectedStyles = ({ selected, theme }) =>
+  selected &&
+  css`
+    label: tag--selected;
+    background-color: ${theme.colors.p500};
+    ${shadowBorder(theme.colors.p500)};
+    color: ${theme.colors.white};
+
+    &:hover {
+      background-color: ${theme.colors.p500};
+      ${shadowBorder(theme.colors.p500)};
```

Comment:
How about staging a darker brand color for hover? A user can click the tag again to unselect, right?

Reply:
I'm not sure if every tag we use is also a toggle. If so, then yes it makes total sense

Reply:
Fair point. How about adding `tagSelectedClickableStyles` to match the `tagClickableStyles` when there's an `onClick handler?